src/components/Menu/MenuItem.tsx

```diff
@@ -110,6 +139,11 @@ class MenuItem extends UIComponent<Extendable<MenuItemProps>, MenuItemState> {
     underlined: PropTypes.bool,
     vertical: PropTypes.bool,
     wrapper: PropTypes.oneOfType([PropTypes.node, PropTypes.object]),
+    menu: customPropTypes.itemShorthand,
+    submenuOpen: PropTypes.bool,
```

Comment:
this one is easy to be confused with `openSubmenu` which we have in the same file. Verb, probably, would help to better differentiate - or past tense could be used. Lets see whether this is consistent with the rest of the code (and naming used), but I would rather like to eliminate this confusion.

Potential suggestions:
- `isSubmenuOpen`
- `submenuOpened`

Reply:
We never use `is` as a prefix for the boolean props, and the similar auto-controlled prop in the Popup, has the name `open`. But, I agree with your concerns, would rather rename it to `submenuOpened`, but not sure as it won't be consistent. What do you think?